core/src/main/java/org/infinispan/remoting/responses/ClusteredGetResponseValidityFilter.java

```diff
@@ -33,8 +33,8 @@ public ClusteredGetResponseValidityFilter(Collection<Address> targets, Address s
    public boolean isAcceptable(Response response, Address address) {
       if (targets.contains(address)) {
          missingResponses--;
-         if (response instanceof SuccessfulResponse) {
-            validResponses++;
+         if (response instanceof SuccessfulResponse || response instanceof ExceptionResponse) {
```

Comment:
Since we are talking about Get commands, I vote for option 1 (ignoring exception and waiting for second response).